1. Basic Web Development/index.html

```diff
@@ -23,63 +22,122 @@
         <div class="square8"></div>
         <div class="square9"></div>
     </div>
-    <div class="preferences hidden">
-        <div class="bar"></div>
-        <div class="content">
-            <h1>Select your preferences:</h1>
-            <div class="section">
-                <h2>Presets:</h2>
-                <form>
-                    <div class="grid-element">
-                        <label for="all">All</label>
-                        <input id="all" name="preset" type="radio" value="All" />
+    <div id="site-content" class="no-display">
+        <div class="begin-arrow">
+            <p>Begin Editing</p>
+            <p> |</p>
+            <p> |</p>
+            <p>\/</p>
+            <a class="to-begin no-decor" href="#create">Edit Pop-up</a>
```

Comment:
One of the requirements was that every route should be able to navigate to every other routes. If you click on this link, you will not have the possibility to return to this view ever again.

Also, this is more of the create pop-up instead of the edit pop-up since that is the view that you will see.